gramex/apps/filemanager/gramex.yaml

```diff
@@ -0,0 +1,28 @@
+import:
+  apps/filemanager/ui:
+    path: $GRAMEXAPPS/ui/gramex.yaml
+    YAMLURL: $YAMLURL/ui/
+
+url:
+  apps/filemanager/home-$*:
+    pattern: /$YAMLURL/
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/index.html
+      template: true
+      filemanager_kwargs: $FILEMANAGER_KWARGS
+      auth: $FILEMANAGER_AUTH
+
+  apps/filemanager/help-$*:
```

Comment:
Keep help separately in help.html. Keep home page separately in index.html. Share the scaffolding through snippets or sub-templates.